### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kartben**
Signed-off-by: Benjamin Cabé <benjamin@eclipse.org>

### CODE REVIEW

**Comment by tobidi** [comment]
Below "## Legal Requirements" (line 18) sentence should start with "Kura is..." rather then "Hono is..."

**Comment by kartben** [comment]
@tobidi sigh... Thanks!!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+First of all, thanks for considering to contribute to Eclipse Kura. We really appreciate the time and effort you want to
+spend helping to improve things around here. And help we can use :-)
+
+Here is a (non-exclusive, non-prioritized) list of things you might be able to help us with:
+
+* bug reports
+* bug fixes
+* improvements regarding code quality e.g. improving readability, performance, modularity etc.
+* documentation (Getting Started guide, Examples, Deployment instructions ni cloud environments)
```
Comment (amitjoy) [inline CONTRIBUTING.md]: A small typo error. Instead of "ni", it should be "in".

**Comment by kartben** [comment]
I wonder if any committer had time to had a look?? Is there something preventing you guys to merge the PR? Do you want some changes to the bullet list of items where help is appreciated?

BTW thanks a lot for the feedback, @tobidi @amitjoy, very cool!

**Pull request merged**